lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -254,6 +313,15 @@ class MockTimers {
 
           nodeTimersPromises.setInterval = this.#realPromisifiedSetInterval;
         },
+        setImmediate: () => {
+          globalThis.setImmediate = this.#realSetImmediate;
+          globalThis.clearImmediate = this.#realClearImmediate;
```

Comment:
it'd probably be wisest to store the original property descriptor, and reapply that, to make sure enumerability etc are all respected?